content/next/index.md

```diff
@@ -78,7 +78,7 @@ The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL
 The description is a short description of the code changes, e.g., _fix: array parsing issue when multiple spaces were contained in string._
 1. A longer commit body MAY be provided after the short description, providing additional contextual information about the code changes. The body MUST begin one blank line after the description.
 1. A footer MAY be provided one blank line after the body.
-  The footer SHOULD contain additional issue references about the code changes (such as the issues it fixes, e.g.,`Fixes #13`).
+  The footer SHOULD contain additional issue references about the code changes (such as the issues it closes, e.g.,`closes #13`).
```

Comment:
I don't think those two words make too much difference. Maybe we can keep both.

Reply:
I would prefer to change this to 

The footer SHOULD contain additional issue references about the code changes (such as the issues it closes, e.g.,`Closes #13`, `Fixes #24` or `Resolves #42`).

Reply:
Which difference are you making between `fixes` and `resolves` ?

Reply:
No differences